cogs/audio.py

```diff
@@ -1876,6 +1876,19 @@ def voice_connected(self, server):
                 vc.audio_player.resume()
 
 
+def check_ffmpeg():
+    """Windows only"""
+    files = ("ffmpeg.exe", "ffplay.exe", "ffprobe.exe")
+    if os.name == "nt":
+        if "ffmpeg" in os.environ['PATH']:
```

Comment:
I rewrote from line ``1883`` to the following,

```python
        try:
            call(['git.exe'], stdout=DEVNULL, stderr=DEVNULL)
            return True
        except:
            return False
```
The only thing that needs to be done besides this is importing ``DEVNULL`` in ``Subprocess``